CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 ############################ BASE ######################################
 
 cmake_minimum_required (VERSION 3.20 FATAL_ERROR)
-project(OpenTurbine LANGUAGES CXX)
+project(OpenTurbine LANGUAGES C CXX)
```

Comment:
Why are you writing this in C?  I see no reason to not do this in C++ like the rest of our code.

Reply:
This was already written in C, please see the issue description.